contrib/unbound-cert-setup.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```

Comment:
Nit, improves portability between environments.

```suggestion
#!/usr/bin/env bash
```